src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -596,3 +600,33 @@ def closure_to_tasklet_sdfg(
     for output in outputs:
         context.body.arrays[output.value.data].transient = False
     return context, outputs
+
+
+def lambda_to_tasklet_sdfg(
```

Comment:
We should ideally unify `lambda_to_tasklet_sdfg` and `closure_to_tasklet_sdfg`, they more or less serve the same purpose. Not sure what's the best approach.

Reply:
Yes, they serve the same purpose, but the representation of the lambda function is different.

Reply:
Addressed in your last commit / discussion below. Closing.